### ISSUE TITLE
Use pydoc's pager module instead of normal print function

### ISSUE THREAD

**Issue by gautamkrishnar** [gautamkrishnar/socli#46]
One of the reedit users suggested that we must use pager instead of print function:
![screenshot 340](https://cloud.githubusercontent.com/assets/8397274/25189669/760df23e-2547-11e7-8f84-ad89f952bcbd.png)

### How to fix
* Comment on this issue if you are interested
* Use pager module (see the sample code in the screenshot)
* Fix:
Windows | Linux | Mac `socli -q` normal mode
Windows `socli -iq` interactive mode
* Open a PR to the develop branch

**Comment by e-beach** [comment]
Yes, I think this is a good idea.
However, there's another issue I'd like to mention. Have you looked at the program https://github.com/gleitz/howdoi? It is very similar to what we are doing, and one big difference is the search url they use, if you look at the source code. I think his system produces better results because it uses Google instead of StackOverflow's search engine.

**Comment by gautamkrishnar** [comment]
Yes. I saw that. Do you think we need to google instead of stack overflow?.

The aim of stack overflow is providing user a command line interface to stack overflow. Not just providing users the ability to search in stackoverflow. That is what makes us different from howdoi.

If we are planning to implement the searching mechanism using google, we will have to rewrite the entire BF soup codes!

**Comment by e-beach** [comment]
If the objective is to provide the best results porssible then I think we should use google.  Honestly, we can just copy with attribution (MIT license) the portion of the code from howdoi that does the search. 
Observe:


Observe the difference:

**High-quality google search results**
![non-retarded method](http://i.imgur.com/hNOwNbp.png)

**Low-quality stack overflow search results:**
![retarded method](http://i.imgur.com/ffrYuIC.png)

**Our method (same as stack overflow search results):**
![our method](http://i.imgur.com/XAZd5WB.png)

Personally, I never use the stack overflow search in the browser for this reason.

**Comment by gautamkrishnar** [comment]
Yes i can see the difference...  We could use google search.... :+1:

**Comment by physicsnerd** [comment]
Hello, I'm interested, and I'm very much a first-timer.

**Comment by gautamkrishnar** [comment]
Thanks a lot @physicsnerd As of now we used another type of implementation for this feature --> see https://github.com/gautamkrishnar/socli/pull/61

But it wont work for windows. So please implement the windows version using the pager module. 🍺

**Comment by sanjitjain2** [comment]
Hello, I'm interested, I am a first timer and would take some time to understand the code. Is this issue still open?
I would like to help.
Can someone guide me?

**Comment by physicsnerd** [comment]
I'm going to assume that I should stop working on the solution I have (it's mostly finished)?

**Comment by gautamkrishnar** [comment]
Sorry @physicsnerd this issue is already fixed partially using interactive mode... We must add pydoc module for windows since windows dont support the interactive mode... As of now you must stop working on the solution because we have some breaking changes to be implemented.... I will inform you once it is ready...

**Comment by gautamkrishnar** [comment]
Added interactive mode.
